build.gradle.kts

```diff
@@ -24,7 +24,36 @@ dependencies {
     testImplementation(platform("org.junit:junit-bom:5.10.0"))
     testImplementation("org.junit.jupiter:junit-jupiter")
 }
+sourceSets {
+    val functionalTest by creating {
+        compileClasspath += sourceSets["main"].output + sourceSets["test"].output
+        runtimeClasspath += sourceSets["main"].output + sourceSets["test"].output
+        java.srcDir(file("src/functionalTest/java"))
+        resources.srcDir(file("src/functionalTest/resources"))
+    }
+}
+
+configurations {
+    val functionalTestImplementation by getting {
+        extendsFrom(configurations["testImplementation"])
+    }
+    val functionalTestRuntimeOnly by getting {
```

Comment:
remove this configuration, since we are not going use runtimeOnly dependencies for now